CHANGELOG.md

```diff
@@ -1,3 +1,243 @@
+### citus v11.0.0_beta (March 22, 2022) ###
+
+* Turns MX on by default
+
+* Adds `citus_finalize_upgrade_to_citus11()` which is necessary to upgrade to
+  Citus 11+ from earlier versions
+
+* Distributes functions with `CREATE FUNCTION` command
```

Comment:
I think `Adds propagation of `CREATE AGGREGATE` commands` could come right after this one. Maybe with the same wording?

Reply:
to conform lexicographical ordering, I placed aggregates before functions here.